app/src/main/java/com/lagradost/cloudstream3/ui/player/CS3IPlayer.kt

```diff
@@ -662,12 +663,7 @@ class CS3IPlayer : IPlayer {
 
         private fun getTrackSelector(context: Context, maxVideoHeight: Int?): TrackSelector {
             val trackSelector = DefaultTrackSelector(context)
-            trackSelector.parameters = DefaultTrackSelector.ParametersBuilder(context)
-                // .setRendererDisabled(C.TRACK_TYPE_VIDEO, true)
-                .setRendererDisabled(C.TRACK_TYPE_TEXT, true)
```

Comment:
This seems to be the culprit and your patch works without the setTrackTypeDisabled
Are there any reasons to have those too?

Reply:
Without those two setTrackTypeDisabled when I choose `No Subtitles` in subtitle selection, player still show subtitle (maybe default subtitle).